exercises/mongo-shell/basic-mongo-queries.md

```diff
@@ -514,33 +661,100 @@ db.movieDetails.find({"genres":["Action", "Adventure"]},
 
 ---
 
-## Group Activities
-During this time the class should split into :five: groups to complete each section.
+## Update Operations
 
-**Activity: :alarm_clock: 15 minutes** 
+The MongoDB query language supports 3 update operations:
+* `updateOne()`
+* `updateMany()`
+* `replaceOne()` - [Read more](https://docs.mongodb.com/manual/reference/method/db.collection.replaceOne/#db.collection.replaceOne) :book: about this operator. 
+
+**Exercise 15** :computer: 
+
+Let's turn our attention towards movies made in 🇮🇳 ! I bet all of you have either heard of or watched "Taare Zameen Par" but unfortunately this collection hasn't got the name right! 
 
-Share with the rest of the class the kind of movies you like by writing 3 interesting queries against this database, at the same time limiting the resulting documents to fields that you find relevant. 
+* Spend **5 minutes** :alarm_clock: to find out what the movie `title` is in our collection! Feel free to use the internet to look up identifying information like the release year or the actor/director/writer names to use in your query filter. 
 
-For example, movies that were part of both the 'Action' and 'Adventure' genres! :boom: 
+* Once you have the name, let's update it to reflect the correct name "Taare Zameen Par". 
 
-Using commands like:
-- `show dbs`
-- `use <database>`
-- `show collections`
+Here's how to perform the update operation:
+```javascript
```

Comment:
I :eyes: what you are trying to do here, you're show syntax, but you need to put that in an explanation somewhere above the query.  This query is not copy/paste ready because of the syntax demo.